src/repart/repart.c

```diff
@@ -7451,7 +7471,7 @@ static int context_fstab(Context *context) {
         if (r < 0)
                 return log_error_errno(r, "Failed to open temporary file for %s: %m", path);
 
-        fprintf(f, "# Automatically generated by systemd-repart\n\n");
+        fprintf(f, "%s\n\n", sd_hdr);
```

Comment:
if these are macros you can output the string as `fputs(AUTOMATIC_FSTAB_HEADER "\n\n", stdout);`, and avoid format strings and their slow interpolation

Reply:
Done